test/integration/narrow/payment-requests.test.js

```diff
@@ -0,0 +1,122 @@
+const db = require('../../../app/data')
+jest.mock('ffc-messaging')
+let createServer
+let server
+let paymentRequest
+
+describe('schemes routes', () => {
+  beforeEach(async () => {
+    await db.sequelize.truncate({ cascade: true })
+
+    await db.scheme.create({
+      schemeId: 1,
+      name: 'SFI'
+    })
+
+    paymentRequest = {
+      completedPaymentRequest: 1,
+      paymentRequestId: 1,
+      schemeId: 1,
+      invoiceNumber: 'InvoiceNumber'
+    }
+
+    createServer = require('../../../app/server')
+    server = await createServer()
+    await server.initialize()
+  })
+
+  afterEach(async () => {
+    await server.stop()
+  })
+
+  test('POST /payment-requests creates new schedule if no schedule', async () => {
```

Comment:
`POST /payment-requests/reset ...`

Reply:
This issue is repeated in all test cases beneath

Reply:
Fixed